ironfish-rust-nodejs/src/structs/transaction.rs

```diff
@@ -245,8 +244,12 @@ impl NativeTransaction {
     /// a miner would not accept such a transaction unless it was explicitly set
     /// as the miners fee.
     #[napi(js_name = "post_miners_fee")]
-    pub fn post_miners_fee(&mut self) -> Result<Buffer> {
-        let transaction = self.transaction.post_miners_fee().map_err(to_napi_err)?;
+    pub fn post_miners_fee(&mut self, spender_hex_key: String) -> Result<Buffer> {
```

Comment:
Consider changing `spender_hex_key: String` to `spender_hex_key: &str` if possible (same for `_post_miners_fee_unchecked`)

Reply:
I think the type here needs to be `String` for napi support